astronomer/providers/snowflake/operators/snowflake.py

```diff
@@ -183,12 +194,13 @@ def execute_complete(
             elif "status" in event and event["status"] == "success":
                 hook = self.get_db_hook()
                 qids = typing.cast(List[str], event["query_ids"])
-                hook.check_query_output(qids)
+                results = hook.check_query_output(qids, self.handler)
                 self.log.info("%s completed successfully.", self.task_id)
+                if self.do_xcom_push:
+                    return results
                 return None
```

Comment:
Why do we need to explicitly write `return None`. What does it return if we don't specify this?

Reply:
good point. we shouldnt return None

Reply:
Yeah initially it was nothing it was just 
```
if self.do_xcom_push:
                    return results
```
mypy was complaining it expecting a return  it tried fixing with optional return for the function, still it is complaining

Reply:
can you paste the error you were getting from mypy? maybe remove the return and get the output from local run of mypy (no need to push)

Reply:
<img width="1002" alt="Screenshot 2022-11-29 at 7 56 34 PM" src="https://user-images.githubusercontent.com/94612827/204555419-6ba63913-b8b3-42a5-b52d-5ab4ae83c7b3.png">

Reply:
if the return type of the execute_complete method is set to Any it is not complaining, if I set that to Optional list of dict or tuple it is complaining

Reply:
Okay 👍🏽